modules/core/src/lib/deck-picker.js

```diff
@@ -318,19 +319,17 @@ export default class DeckPicker {
   // Calculate a picking rect centered on deviceX and deviceY and clipped to device
   // Returns null if pixel is outside of device
   getPickingRect({deviceX, deviceY, deviceRadius, deviceWidth, deviceHeight}) {
-    const valid = deviceX >= 0 && deviceY >= 0 && deviceX < deviceWidth && deviceY < deviceHeight;
+    // Create a box of size `radius * 2 + 1` centered at [deviceX, deviceY]
+    const x = Math.max(0, deviceX - deviceRadius);
+    const y = Math.max(0, deviceY - deviceRadius - 1);
```

Comment:
Why `y` is calculated differently than `x` ?

Reply:
device coordinates starts from bottom-left.